src/main/java/net/datafaker/providers/sport/Baseball.java

```diff
@@ -0,0 +1,59 @@
+package net.datafaker.providers.sport;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+/**
+
+ * Generate random components of baseball game, e.g. teams, coaches, positions and players.
+ *
+ * @author unknown and irakatz
+ * @since 0.8.0
```

Comment:
Small issue here with the version number, this should be 1.7.0 I think?

Reply:
Yeah, I mean 1.7.0. I'll change the version number.